frontend/src/component/insights/Insights.tsx

```diff
@@ -77,23 +76,65 @@ export const Insights: VFC = () => {
                     }
                 />
             </StickyWrapper>
-            <ConditionallyRender
-                condition={isInsightsV2Enabled}
-                show={
-                    <InsightsCharts
-                        loading={loading}
-                        projects={projects}
-                        {...insightsData}
-                    />
-                }
-                elseShow={
-                    <LegacyInsightsCharts
-                        loading={loading}
-                        projects={projects}
-                        {...insightsData}
-                    />
-                }
+            <LegacyInsightsCharts
+                loading={loading}
+                projects={projects}
+                {...insightsData}
+            />
+        </>
+    );
+};
+
+const NewInsights = () => {
+    const [scrolled, setScrolled] = useState(false);
+
+    const stateConfig = {
+        projects: FilterItemParam,
+        from: FilterItemParam,
+        to: FilterItemParam,
+    };
+    const [state, setState] = usePersistentTableState('insights', stateConfig);
+    const { insights, loading, error } = useInsights(
+        state.from?.values[0],
+        state.to?.values[0],
+    );
+
+    const projects = state.projects?.values ?? [allOption.id];
```

Comment:
projects have values and operator due to the original design of our flexible filters. We need to extract values from the object. Keeping original * as a default option since useInsightsData uses it for client side filtering of projects.